lib/core/src/server/dev-server.ts

```diff
@@ -85,24 +85,25 @@ async function useStatics(router: any, options: { staticDir?: string[] }) {
   if (staticDir && staticDir.length) {
     await Promise.all(
       staticDir.map(async (dir) => {
-        const [currentStaticDir, staticEndpoint] = dir.split(':').concat('/');
-        const localStaticPath = path.resolve(currentStaticDir);
-
-        if (!(await pathExists(localStaticPath))) {
-          logger.error(`Error: no such directory to load static files: ${localStaticPath}`);
-          process.exit(-1);
+        const [dirname, location = '/'] = dir.split(':');
+        const dirpath = path.resolve(dirname);
+        const endpoint = location.startsWith('/') ? location : `/${location}`;
+
+        if (!(await pathExists(dirpath))) {
+          logger.warn(`Failed to load static files, no such directory: ${dirpath}`);
+          logger.warn(`Check your \`-s\` or \`--static-dir\` option for start-storybook.`);
```

Comment:
Could we not make a suggestion instead to create the directory as well?

Reply:
Done